test/check-machines-networks

```diff
@@ -312,6 +312,8 @@ class TestMachinesNetworks(VirtualMachinesCase):
                     b.wait_not_present(f"#network-{self.name}-{connectionName}-ipv4-address")
                     b.wait_not_present(f"#network-{self.name}-{connectionName}-ipv6-address")
 
+                self.test_obj.toggleNetworkRow(self.name, connectionName)
```

Comment:
So this is needed because of the pkg/lib update.  The implementation of the ListingTable changed recently. Now the expanded state of each row is saved in the LisingComponent as a state variable. https://github.com/cockpit-project/cockpit/blob/main/pkg/lib/cockpit-components-table.jsx#L77

The problem is that it relies on the supplied row Id to store if a row is expanded or not. 

What the test does is, create network -> expand the row -> (now the row is expanded and this is stored in ListingTable state) -> delete network -> create again network with the same name / id. The problem here is that the new network will be expanded because of this name colision.

This is a bug in the ListingTable implementation, or we can also workaround it here with using uuid instead of name. 

But it's not a big deal, for users it's not really harmful, but the tests caught it when this line was missing. 

If you think this need to be fixed / changed within this PR i can do it on monday.